firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/internal/InstallActivity.java

```diff
@@ -143,6 +143,7 @@ private void startAndroidPackageInstallerIntent() {
     }
 
     intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+    intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
```

Comment:
Because these flags are bits, they can be combined like:
```
intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
```

See examples [here](https://source.corp.google.com/search?q=FLAG_ACTIVITY_NEW_TASK.*FLAG_ACTIVITY_CLEAR_TASK&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental).

Reply:
Ah i see. good thought.